.github/workflows/code_ext.yml

```diff
@@ -0,0 +1,82 @@
+name: Code Extention CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - 'master'
+      - 'dev'
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: npm install
+        run: |
+          npm ci --ignore-scripts
+          cd client
+          npm ci
+          cd ..
+          cd server
+          npm ci
+          cd ..
```

Comment:
The server submodule also uses a `postinstall` script to install its submodules, so once we
```
cd server
npm ci --ignore-scripts
```
we also have to
```
cd antlr4-tsplang
npm ci --ignore-scripts
cd ../tsplang-plugins
npm ci --ignore-scripts
cd ..
```